mitosheet/src/utils/formatColumns.tsx

```diff
@@ -1,346 +0,0 @@
-                            numDecimals = cellDataToParse.toString().split(".")[1].length || 0;
-                        }
-                        return formatCellDataAsStringWithCommas(cellData, numDecimals)
-                    }
-                case FormatType.PLAIN_TEXT:
-                    return '' + cellData
-                case FormatType.ROUND_DECIMALS: {
-                    const numDecimals = columnFormatType.numDecimals
-                    return Number(cellData).toLocaleString("en-US", {minimumFractionDigits: numDecimals, maximumFractionDigits: numDecimals})
-                }
-                case FormatType.PERCENTAGE: {
-                    return Number(cellData).toLocaleString("en-US", {style: 'percent', minimumFractionDigits: 2, maximumFractionDigits: 2})
-                }
-                case FormatType.ACCOUNTING: {
-                    return Number(cellData).toLocaleString("en-US", {style: "currency", currency: "USD", currencySign: "accounting"})
-                }
-                case FormatType.K_M_B: {
-                    // Ensure we're operating on a number.
-                    const number = Number.parseFloat(String(cellData))
-                    const number_abs = Math.abs(number)
-                    if (number_abs >= 1000000000) {
-                        return (number / 1000000000).toFixed(1).replace(/\.0$/, '') + 'B';
-                    } else if (number_abs >= 1000000) {
-                        return (number / 1000000).toFixed(1).replace(/\.0$/, '') + 'M';
-                    } else if (number_abs >= 995) {
-                        return (number / 1000).toFixed(1).replace(/\.0$/, '') + 'K';
-                    } else {
-                        // We make anything less than ABS(995) displayed as 0k to match Excel formatting 
-                        return '0k';
-                    }
-                }
-                case FormatType.SCIENTIFIC_NOTATION:
-                    return Number.parseFloat(String(cellData)).toExponential(2);
-            }
-        } else {
-            return '' + cellData
-        }
-    } catch {
-        return '' + cellData 
-    }
-    
-}
-
-/* 
-    Sends the changeColumnFormat message to the backend so that the format of the selected columns
-    will be updated in the state.
-*/
-export const changeFormatOfSelectedColumns = async (
-    sheetIndex: number,
-    selections: MitoSelection[], 
-    newFormatTypeObj: FormatTypeObj, 
-    sheetData: SheetData | undefined,
-    mitoAPI: MitoAPI
-): Promise<void> => {
-    
-    const numberColumnIDsSelected = getSelectedNumberSeriesColumnIDs(selections, sheetData)
-    
-    await mitoAPI.editChangeColumnFormat(
-        sheetIndex,
-        numberColumnIDsSelected,
-        newFormatTypeObj
-    )
-}
-
-/* 
-    Change the format of a single columnID
-*/
-export const changeFormatOfColumnID = async (sheetIndex: number, columnID: string, newFormatTypeObj: FormatTypeObj, mitoAPI: MitoAPI): Promise<void> => {
```

Comment:
This shouldn't be a function! It literally just calls another function, with pretty much the same params.